Warn that cloned templates are not automatically updated (#2740)
elastic/docs-content: solutions/observability/apm/index-lifecycle-management.md

## Namespace-level index lifecycle policies [apm-data-streams-custom-policy-namespace]

> It is also possible to create more granular index lifecycle policies that apply to individual namespaces. This process is similar to the above tutorial, but includes cloning and modify the existing index template to use a new `*@custom` component template.

Reviewer: ```suggestion
It is also possible to create more granular index lifecycle policies that apply to individual namespaces. This process is similar to the above tutorial, but includes cloning the original index template(s) and use the index lifecycle you've created in it.
```
